buildModels.js

```diff
@@ -0,0 +1,29 @@
+var fs = require('fs');
```

Comment:
I'm a touch more a fan of underscores instead of camel case for filenames, but it's your call. =P